<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>592</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>592</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>592</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KickRender</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_load">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Scene</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>761</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Scene Settings</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_TexPath">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>531</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Texture Search Path: </string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_selectTexPath">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>741</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Output path Replacement</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>From:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>To:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_OutPathFrom">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>661</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_OutPathTo">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>601</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_selectOutPathTo">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>50</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>761</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Kick Settings</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>kick Path:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_selectKickPath">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_KickPath">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>601</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>441</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Disable render window (recommended for batch rendering)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>441</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Disable progressive rendering (recommended for batch rendering)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_SkipLicenseCheck">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>431</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Skip license check (assume license is not available)</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_Render">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>761</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Render</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_log">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>391</width>
      <height>541</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::NoWrap</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="backgroundVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_seqPath">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_StopRender">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>761</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Start:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>End:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_startFrame">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>- F</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_endFrame">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>- F</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Total:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_totalFrame">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>- F</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
